### PR TITLE
Add info to R readme

### PR DESCRIPTION
**Pull request by aerlaut**
# Description
Add info to setup the R worker environment

# Details
#### URL to issue
N/A

#### Link to staging deployment URL (or set N/A)
N/A

#### Links to any PRs or resources related to this PR
N/A

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [x] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: r/README.md [hunk 1]
```diff
@@ -1,56 +1,68 @@
...
-Development is done inside a development container that is automatically built,
-run, and managed by Visual Studio Code. You do not need R, R Studio, or a Python
-virtual environment to be installed locally.
+    # Enter the R REPL
+    $ R
 
-You must have the[Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension installed, as specified
-in the main README.
+    > R.Version()$version.string
+    [1] "R version 4.0.5 (2021-03-31)"
```
Comment (gerbeldo) [inline r/README.md]: `R --version` in a shell works too

File: r/README.md [hunk 2]
```diff
@@ -1,56 +1,68 @@
...
 
-You should be prompted to run the workspace inside a container. Accept this. Once
-you see the folder structure, the worker is running and you have access to the
-R worker's container.
+If you have a different version of R, please uninstall the installation. Once the package is uninstalled,download the required R version from the [CRAN hosted site](https://cran.r-project.org/bin/). Using the Homebrew distribution of R is not recommended.
```
Comment (gerbeldo) [inline r/README.md]:
```suggestion
If you have a different version of R, please uninstall it, using the instructions in the [R manual](https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Uninstalling-under-macOS). Once the package is uninstalled, download the required R version from the [CRAN hosted site](https://cran.r-project.org/bin/). For macOS, the required binaries are [here](https://cran.r-project.org/bin/macosx/base/). **Using the Homebrew distribution of R is not recommended.**
```

File: r/README.md [hunk 3]
```diff
@@ -1,56 +1,68 @@
...
+    renv::restore()
 
-``` R
-renv::restore()
-```
+Most of the development is done using RStudio. To
+
+### Development with Visual Studio Code
+
+Aside from RStudio, development can also be done inside a development container that is automatically built, run, and managed by Visual Studio Code. You must have the [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension installed, as specified in the main README.
```
Comment (gerbeldo) [inline r/README.md:36]: Maybe add link to main README?

**Review by gerbeldo — APPROVED** [review]

File: r/README.md [hunk 4]
```diff
@@ -1,56 +1,68 @@
...
 - Runs the task function specified in the request from the Python worker.
 - Returns the results back to the Python worker.
-Setup
------
 
-Open the r workspace:
+## Setup
 
-    code r.code-workspace
+The R worker runs uses R version 4.0.5. To check your R version, you can run the following command
```
Comment (gerbeldo) [inline r/README.md]:
```suggestion
The R worker (as of Jan 2022) uses R version **4.0.5**. To check your R version, you can run the following command
```

File: r/README.md (hunk 2 above)
Comment (aerlaut) [reply r/README.md]: changed

File: r/README.md (hunk 4 above)
Comment (aerlaut) [reply r/README.md]: Added

File: r/README.md (hunk 3 above)
Comment (aerlaut) [reply r/README.md:36]: I think we should improve on these READMEs (main, python and R) to make the distinction between these different READMEs clearer.

**Pull request merged**